packages/block-editor/src/hooks/style.js

```diff
@@ -96,6 +97,22 @@ function addAttribute( settings ) {
 	return settings;
 }
 
+/**
+ * Filters a style object returning only the keys
+ * that are serializable for a given block.
+ *
+ * @param {Object} style Input style object to filter.
+ * @param {Object} blockSupports Info about block supports.
+ * @return {Object} Filtered style.
+ */
+export function omitNonSerializableKeys( style, blockSupports ) {
```

Comment:
`omitNonSerializableKeys` sounds like these keys _cannot_ be serialized. Would a better name be `omitSerializationSkippedKeys`? Not sure :)

Reply:
what about `omitKeysNotToSerialize`?

Reply:
👍